movie/src/components/ActionContainer/SortCard/SortOption.json

```diff
@@ -0,0 +1,10 @@
+[
+    { "id": 1, "label": "Popularity Descending", "value": "p_desc" },
```

Comment:
i think the `value` name could be a bit hard to understand. Would be grate if you can change to something that is much easy to understand and readable